NuGet.Config

```diff
@@ -9,6 +9,10 @@
     <add key="nuget-build" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
     <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/msft_consumption%40Local/nuget/v3/index.json" />
   </packageSources>
+  <auditSources>
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Does this cause some static validations to raise false positives about nuget.org usage?

Reply:
nothing in the devdiv private build: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=9962868&view=results